src/persist-client/src/fetch.rs

```diff
@@ -44,6 +46,30 @@ use crate::internal::state::{BatchPart, HollowBatchPart};
 use crate::read::LeasedReaderId;
 use crate::ShardId;
 
+/// The columnar format we write down a batch with.
```

Comment:
nit: copy-pasta on the rustdoc? seems like we could make it pub(crate) and then remove the rustdoc

Reply:
done